### DISCUSSION TITLE
Upgrading to recent versions of Mkdocs (>1.4+) and materials breaks EzLinks plugin

### DISCUSSION THREAD

**Discussion by s10doc** [squidfunk/mkdocs-material#6178] [General]
It would seem the the [mkdocs-ezlink-plugin](https://github.com/orbikm/mkdocs-ezlinks-plugin) has discontinued its upgrades and has remained at version **0.1.14**

Unfortunately, when I upgrade to a recent version of Mkdocs, all the links are broken because the ezlinks plugin no longer works.   

I'm currently using mkdocs-material **8.2.8+insiders.4.12.0**

Does anyone know if there is a suitable plugin that can function in the same or similar way to ezlinks and is compatible with Mkdocs and Materials?

**Comment by alexvoss** [comment]
Depends a bit on what you need specifically but I recently came across the [Wikilinks extension](https://python-markdown.github.io/extensions/wikilinks/) to Python Markdown. Put this in your `mkdocs.yml`:

```
markdown_extensions:
  - wikilinks
```
